app/react/Settings/components/NavlinkForm.js

```diff
@@ -90,58 +99,118 @@ export class NavlinkForm extends Component {
+                      </button>
                     </div>
                   </div>
+                  <div className="row">
+                    <ShowIf if={link.type === 'group'}>
+                      <div style={{ paddingLeft: '80px' }}>
+                        <div className="row">
+                          <div className="col-sm-12">
+                            {links[index].sublinks?.map((_, i) => (
+                              <div
+                                className="row"
+                                style={{ paddingBottom: '5px', paddingTop: '5px' }}
+                              >
+                                <div className="col-sm-3" style={{ display: 'flex' }}>
+                                  <span style={{ padding: '5px 10px 0px 0px' }}>
+                                    <Icon icon="link" />
+                                  </span>
+                                  <div className={`${titleClass} input-group-width`}>
+                                    <span className="input-group-addon">Title</span>
+                                    <Field
+                                      model={`settings.navlinksData.links[${index}].sublinks[${i}].title`}
+                                    >
+                                      <input className="form-control" style={{ width: '100%' }} />
+                                    </Field>
+                                  </div>
+                                </div>
+                                <div className="col-sm-8">
+                                  <div className="input-group">
+                                    <span className="input-group-addon">URL</span>
+                                    <Field
+                                      model={`settings.navlinksData.links[${index}].sublinks[${i}].url`}
+                                    >
+                                      <input className="form-control" />
+                                    </Field>
+                                  </div>
+                                </div>
+                                <div className="col-sm-1" style={{ paddingLeft: '0px' }}>
+                                  <button
+                                    type="button"
+                                    className="btn btn-danger btn-xs property-remove, menu-delete-button"
+                                    onClick={() => this.props.removeGroupLink(index, i)}
+                                  >
+                                    <Icon icon="trash-alt" /> Delete
+                                  </button>
+                                </div>
+                              </div>
+                            ))}
+                            <div className="row">
+                              <div className="col-sm-12">
+                                <button
+                                  className="menu-link-group-button"
+                                  type="submit"
```

Comment:
I think having this as `type="submit"` is causing an issue where when you add a link inside a group it automatically saves.

Reply:
Removed